scripts/setup/create_multinode.sh

```diff
@@ -100,31 +96,24 @@ function setup_master() {
 
     clone_loader $MASTER_NODE
 
-    MN_CLUSTER="./vhive/scripts/cluster/create_multinode_cluster.sh ${OPERATION_MODE}"
+    server_exec $MASTER_NODE 'sudo wget -q https://github.com/mikefarah/yq/releases/download/v4.27.3/yq_linux_amd64 -O /usr/bin/yq && sudo chmod +x /usr/bin/yq'
+    server_exec $MASTER_NODE '~/loader/scripts/setup/rewrite_yaml_files.sh'
+
+
+    MN_CLUSTER="pushd ~/vhive/scripts > /dev/null && ./setup_tool create_multinode_cluster ${OPERATION_MODE} && popd > /dev/null"
     server_exec "$MASTER_NODE" "tmux send -t master \"$MN_CLUSTER\" ENTER"
 
     # Get the join token from k8s.
-    while [ ! "$LOGIN_TOKEN" ]
-    do
+    while ! server_exec "$MASTER_NODE" "[ -e 'vhive/scripts/masterKey.yaml' ]"; do
```

Comment:
Everywhere around we are using absolute paths like `~/vhive/scripts`. Please use it here and on 116th line as well.